frontend/test/pytest/test_quantum_control.py

```diff
@@ -454,7 +454,7 @@ def native_controlled():
         return qml.state()
 
     compiled = qjit()(native_controlled)
-    assert all(sign in compiled.mlir for sign in ["ctrls", "ctrlvals"])
+    assert all([sign in compiled.mlir for sign in ["ctrls", "ctrlvals"]])
```

Comment:
Does all not work on generators?

Reply:
I think what happened is that some other error got in the way and I did this to solve it. But it should be reverted back.